packages/components/forms/stories/RadioButtonField.stories.tsx

```diff
@@ -22,7 +21,7 @@ export default {
 export const Basic = (args: RadioButtonFieldProps) => {
   const [activeOption, setActiveOption] = useState('yes');
   return (
-    <FieldGroup>
+    <Flex flexDirection="column">
```

Comment:
For some reason, FieldGroup and Form is not working. I don't want to lose too much time in fixing it, since we will remove it anyway.

Reply:
I'm using `Flex` in `TextArea` as well